pkg/retistruct/R/server.R

```diff
@@ -356,6 +358,15 @@ server <- function(input, output, session) {
   
   # phi0 handler
   observeEvent(input$phi0, {
+    if (!is.numeric(input$phi0)) {
```

Comment:
I'm not sure this is working quite as I'd expect, but let's leave it for now, and do any work on improving in connection with #64